### PR TITLE
add arguments `file` is boolean，`path` is srting 

### PR DESCRIPTION
**Pull request by Codelone**
Sometimes， we need to use a variable path to execute commands, so replace the file path of PARQUET、ORC、 JSON、CSV  by adding arguments。
if need use ，need to set  " -f  -p param1=/home/xxx.csv,param2=/home/xxx.csv " 
then tag/edge path in application.conf needs to be set to param1, param2.
if don't need it , don't use " -f  -p ".

### CODE REVIEW

**Comment by wey-gu** [comment]
Great improvement~~
Maybe we need some docs/examples as well :)

**Comment by Codelone** [comment]
> 
In practical using, we will create a scheduler task, the `csv` file path is fixed, but our file name maybe  changing, such as a date suffix, so we need to use a variable path.

File: exchange-common/src/main/scala/com/vesoft/exchange/common/Package.scala [hunk 1]
```diff
@@ -97,4 +97,6 @@ final case class Argument(config: String = "application.conf",
                           hive: Boolean = false,
                           directly: Boolean = false,
                           dry: Boolean = false,
-                          reload: String = "")
+                          reload: String = "",
+                          file: Boolean = false,
+                          path: String = "")
```
Comment (Nicole00) [inline exchange-common/src/main/scala/com/vesoft/exchange/common/Package.scala:102]: it's a great work, thanks!
maybe we should use more understandable variable name，how about change `file` to `variable`， `path` to `param`?

**Comment by Nicole00** [comment]
this pr look good to me,  and the code needs to be formatted please. @Codelone

**Comment by Codelone** [comment]
It has been changed，please check it。

**Comment by Nicole00** [comment]
I just noticed there's no test for the variable path for config file, could you please add some test to make sure it works?

**Comment by Codelone** [comment]
test case method `configsWithVariableSuite`

**Review by Nicole00 — APPROVED** [review]
Great job！

**Pull request merged**